indico/modules/categories/operations.py

```diff
@@ -97,3 +102,19 @@ def _log_category_update(category, changes):
                 what = what['title']
         category.log(CategoryLogRealm.category, LogKind.change, 'Category', f'{what} updated', session.user,
                      data={'Changes': make_diff_log(changes, log_fields)})
+    logger.info('Category %s updated by %s', category, session.user)
+
+
+def update_event_move_request(request, accept, reason=None):
+    request.state = MoveRequestState.rejected
+    request.moderator_comment = reason
+    request.moderator = session.user
+    if accept:
+        request.state = MoveRequestState.accepted
+        request.event.move(request.category)
+    else:
+        category = request.category
+        sep = ' \N{RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK} '
+        category.log(CategoryLogRealm.category, LogKind.negative, 'Category', 'Move request rejected', session.user,
```

Comment:
`CategoryLogRealm.events` and "Moderation" instead of "Category" for the module name feels more appropriate here